libs/datamodel/connectors/dml/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "dml"
+version = "0.1.0"
+authors = ["Carmen Berndt <carmen@berndt-home.de>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+uuid = {version = "0.8", features = ["serde", "v4"]}
+cuid = {git = "https://github.com/prisma/cuid-rust"}
+prisma-value = {path = "../../../prisma-value"}
+chrono = {version = "0.4.6", features = ["serde"]}
+serde = { version = "1.0.90", features = ["derive"] }
+serde_json = "1.0"
```

Comment:
This dependency might not be needed?

Reply:
it's needed for `scalars.rs`

Reply:
Ah I see. I only skimmed the existing `dml` code and didn't find something. Makes sense 👍